docs/src/snap/reference/bootstrap-config-reference.md

```diff
@@ -302,7 +302,8 @@ Can be used to point to an external datastore like etcd.
 **Type:** `list[string]` <br>
 **Required:** `No` <br>
 
-The server addresses to be used when `datastore-type` is set to `external`.
+The server addresses, in URL form,
```

Comment:
The url form does not require the protocol: https://stackoverflow.com/questions/59600332/is-a-protocol-eg-http-or-https-required-for-a-url-to-be-valid

Maybe write the explicit format or an example instead.